core/domain/user_services.py

```diff
@@ -359,7 +359,7 @@ def get_users_settings(user_ids):
             result.append(UserSettings(
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
-                role=feconf.ROLE_ID_ADMIN,
+                role=feconf.ROLE_ID_EXPLORATION_EDITOR,
```

Comment:
Actually this "user" is responsible for doing exploration migrations etc. -- I think they might need to have rights to edit any exploration (incl. private ones), at least. Any reason to not be admin?